Source/WebCore/dom/TrustedType.h

```diff
@@ -50,4 +51,5 @@ WEBCORE_EXPORT ExceptionOr<String> trustedTypeCompliantString(TrustedType, Scrip
 
 WEBCORE_EXPORT ExceptionOr<String> requireTrustedTypesForPreNavigationCheckPasses(ScriptExecutionContext&, const String& urlString);
 
+WEBCORE_EXPORT String getAttributeTrustedType(const String& tagName, const String& attributeParameter, const String& elementNamespace, const String& attributeNamespace, String& sink);
```

Comment:
nit: getTrustedTypeForAttribute?

Not sure if that's better or worse just an idea.

Reply:
Done. However, we still have the build error in Mac. Not sure if this is related.